packages/rum/src/domain/segmentCollection/segmentCollection.ts

```diff
@@ -89,19 +88,11 @@ export function doStartSegmentCollection(
     flushSegment('view_change')
   })
 
-  const { unsubscribe: unsubscribeBeforeUnload } = lifeCycle.subscribe(LifeCycleEventType.BEFORE_UNLOAD, () => {
-    flushSegment('before_unload')
-  })
-
-  const { stop: unsubscribeVisibilityChange } = addEventListener(
-    emitter,
-    DOM_EVENT.VISIBILITY_CHANGE,
-    () => {
-      if (document.visibilityState === 'hidden') {
-        flushSegment('visibility_hidden')
-      }
-    },
-    { capture: true }
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏should we keep using capture?

Reply:
Ah, good catch. I think that it should not have a big impact, but we should use `capture` most of the time to avoid being impacted by the app stopping the event propagation. In the long run I'd be in favor of using capture by default for all listeners.